scripts/send-trigger-requests.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash +x
```

Comment:
To avoid [SC2096](https://www.shellcheck.net/wiki/SC2096):
```suggestion
#!/usr/bin/env bash
```